packages/core/src/oidc/init.ts

```diff
@@ -198,6 +209,66 @@ export default function initOidc(envSet: EnvSet, queries: Queries, libraries: Li
       },
     },
     extraParams: [OIDCExtraParametersKey.InteractionMode],
+    extraTokenClaims: async (ctx, token) => {
+      if (!EnvSet.values.isDevFeaturesEnabled) {
+        return;
+      }
+
+      const isTokenClientCredentials = token instanceof ctx.oidc.provider.ClientCredentials;
+
+      /**
+       * Cloud connection should not go through this custom JWT logic.
+       */
+      if (isTokenClientCredentials) {
+        const { appId } = await cloudConnection.getCloudConnectionData();
+        if (token.clientId === appId) {
+          return;
+        }
+      }
+
+      const {
+        value: { script, envVars },
+      } = (await trySafe(
```

Comment:
wondering why the return type is set to { value:  XXXX }, can we format the return value within the `logtoConfigs.getJwtCustomizer` method?